pkgs/tools/graphics/piglit/default.nix

```
@@ -0,0 +1,79 @@
+{ stdenv
+, fetchFromGitLab
+, lib
+, cmake
+, freeglut
+, libGL
+, libGLU
+, libglvnd
+, makeWrapper
+, ninja
+, pkg-config
+, python3Packages
+, waffle
+, wayland
+, libX11
+, libXrender
+, libxcb
+, libxkbcommon
+}:
+
+stdenv.mkDerivation rec {
+  pname = "piglit";
+  version = "unstable-2020-10-23";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "mesa";
+    repo = "piglit";
+    rev = "59e695c16fdcdd4ea4f16365f0e397a93cef7b80";
+    sha256 = "kx0+2Sdvdc3SbpAIl2OuGCWCpaLJC/7cXG+ZLvf92g8=";
+  };
+
+  buildInputs = [
+    freeglut
+    libGL
+    libGLU
+    libglvnd
+    libX11
+    libXrender
+    libxcb
+    libxkbcommon
+    waffle
+    wayland
+  ] ++ (with python3Packages; [
+    numpy
+    Mako
+  ]);
```

It's better to wrap the interpreter with the expected modules.

```suggestion
    (python3.withPackages (ps: with ps; [
      Mako
      numpy
    ]))
  ];
```

(This should probably be sorted before `waffle` to keep alphabetical order.)